Dockerfile

```diff
@@ -48,19 +36,9 @@ RUN \
   && cat *aerospike-tools*.sha256 | cut -d' ' -f1) \
   && echo "$TOOLS_SHA256 *aerospike-tools.tgz" | sha256sum -c -
 
-FROM debian:bullseye-slim as install
-
-# Work from /aerospike
-WORKDIR /aerospike
-ENV PATH /aerospike:$PATH
-
-# Install Aerospike
-
-COPY --from=build Python-* /aerospike/Python
-COPY --from=build aerospike/aerospike-tools*.deb /aerospike/aerospike/
-
-RUN apt update && apt install -y libreadline8 make && ls /aerospike && ls /aerospike/Python && make -C Python install && dpkg -i /aerospike/aerospike/aerospike-tools*.deb \
-  && apt-get purge -y --auto-remove make &&  rm -rf aerospike /var/lib/apt/lists/*
+# Install Aerospike tools
+RUN apt update && apt install -y libreadline8 && ls /aerospike && dpkg -i /aerospike/aerospike/aerospike-tools*.deb \
```

Comment:
Second install of libreadline

Reply:
added back multi stage build, this is only done in the install stage now